docs/en/reference/events.rst

```diff
@@ -215,6 +215,7 @@ specific to a particular entity class's lifecycle.
         <?php
         use Doctrine\DBAL\Types\Types;
         use Doctrine\ORM\Event\PrePersistEventArgs;
+        use Doctrine\ORM\Mapping\{Entity, HasLifecycleCallbacks, PrePersist, PreUpdate};
```

Comment:
To fit our coding style:

```suggestion
        use Doctrine\ORM\Mapping\Entity;
        use Doctrine\ORM\Mapping\HasLifecycleCallbacks;
        use Doctrine\ORM\Mapping\PrePersist;
        use Doctrine\ORM\Mapping\PreUpdate;
```

Reply:
Fixed in https://github.com/doctrine/orm/pull/10916/commits/d02b5bcf5d82762c54c4a48f0bac0857559e43ce